packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -54,34 +55,65 @@ export default {
   tags: ["autodocs"],
 } as Meta<typeof Tooltip>;
 
+const proficiencyChoices = [
```

Comment:
@JFChoe could we inline this into the example? Seems like it's only used once

Reply:
fixed!